plugins/carbon_black_cloud/icon_carbon_black_cloud/connection/connection.py

```diff
@@ -82,6 +82,8 @@ def _handle_response(
             except JSONDecodeError as json_error:
                 error_data = f"Invalid JSON response: {response.content}"
                 error_cause = json_error
+        elif response.status_code == 204:
```

Comment:
My bad missing this on a previous release! There's a plugin changes required to make this a release, need to bump the version and history in plugin spec, and run the `insight-plugin refresh` to apply this to the necessary files

Reply:
@ekelly-r7 , can you add this into your PR for the CB page size updates and we can bump the versioning etc in it and close this one out?

Reply:
Yep no problem, I'll add this in with the work being completed under https://rapid7.atlassian.net/browse/SOAR-17314